virtualbox/build-centos-iso/new-iso.sh

```diff
@@ -8,10 +8,23 @@ OS_NAME="centos7"
 ##
 ## Add public key
 ##
-add_ssh_key(){
-    echo "#### 1. Add ssh public key and create user-data file"
+get_host_timezone(){
+  if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    echo "$(cat /etc/timezone)"
+  elif [[ "$OSTYPE" == "darwin"* ]]; then
+    echo "$(systemsetup -gettimezone)"
```

Comment:
on macos, admin right is required to execute the command `You need administrator access to run this tool... exiting!`.